src/autopas/selectors/tuningStrategy/TuningStrategyFactory.cpp

```diff
@@ -95,23 +99,24 @@ std::unique_ptr<autopas::TuningStrategyInterface> autopas::TuningStrategyFactory
         return nullptr;
       }
 
-      tuningStrategy =
-          std::make_unique<FullSearch>(allowedContainers, allowedCellSizeFactors.getAll(), allowedTraversals,
-                                       allowedLoadEstimators, allowedDataLayouts, allowedNewton3Options);
+      tuningStrategy = std::make_unique<FullSearch>(allowedContainers, allowedCellSizeFactors.getAll(),
+                                                    allowedTraversals, allowedLoadEstimators, allowedDataLayouts,
+                                                    allowedNewton3Options, allowedVerletRebuildFrequencies.getAll());
       break;
     }
 
     case TuningStrategyOption::bayesianSearch: {
-      tuningStrategy = std::make_unique<BayesianSearch>(allowedContainers, allowedCellSizeFactors, allowedTraversals,
-                                                        allowedLoadEstimators, allowedDataLayouts,
-                                                        allowedNewton3Options, maxEvidence, acquisitionFunctionOption);
+      tuningStrategy = std::make_unique<BayesianSearch>(
+          allowedContainers, allowedCellSizeFactors, allowedTraversals, allowedLoadEstimators, allowedDataLayouts,
+          allowedVerletRebuildFrequencies, allowedNewton3Options, maxEvidence, acquisitionFunctionOption);
       break;
     }
 
     case TuningStrategyOption::bayesianClusterSearch: {
       tuningStrategy = std::make_unique<BayesianClusterSearch>(
           allowedContainers, allowedCellSizeFactors, allowedTraversals, allowedLoadEstimators, allowedDataLayouts,
-          allowedNewton3Options, maxEvidence, acquisitionFunctionOption, outputSuffix);
+          allowedNewton3Options, allowedVerletRebuildFrequencies.getAll(), maxEvidence, acquisitionFunctionOption,
```

Comment:
Why is `getAll()` used unlike `allowedCellSizeFactors`?

Reply:
because i use set in the Interface. As mentioned before it didn't work with Numberset